bbc/simorgh
# WSTEAM1-963 - Render social embeds on CAF pages

Resolves https://jira.dev.bbc.co.uk/browse/WSTEAM1-963

Overall changes
======
- Add check for older CPS assets with social embeds to find the correct social embed block
- Adds temporary check in CaptionWrapper component to ensure that the Optimo styling is applied for CAF enabled services. This will be removed and default to Optimo styling once all services are moved to CAF
- Adds a global const file to keep track of CAF enabled services

Code changes
======
- Updates Optimo SocialEmbed component to check for additional `oEmbed` location in page data
- Adds temporary additional service checks to enable Optimo stylings for CAF enabled services in social embeds

Testing
======
1. Update the `cafServices.const.ts` file and add 'mundo' to the array list
2. Visit http://localhost:7080/mundo/23263889?renderer_env=caftest
3. Scroll down and confirm that the Twitter and Instagram embeds render and work (note that the YouTube one has malformed data)
4. Update the `cafServices.const.ts` file and add 'portugese' to the array list
5. Visit http://localhost:7080/portuguese/brasil-59057279?renderer_env=caflive
6. Scroll to the bottom of the article and confirm that the Youtube embeds render and work

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
